### PR TITLE
Clean up temp files in Dockerfile to save image size

### PR DESCRIPTION
**Pull request by PeterDaveHello**
#### Summary

Clean up temporary/cache files in Dockerfile to save image size

Result:

```text
REPOSITORY            TAG             IMAGE ID            CREATED              SIZE
after                 latest          521681716ad5        About a minute ago   107MB
before                latest          c192efaae426        3 minutes ago        147MB
```

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -2,11 +2,13 @@ FROM ubuntu:20.04
 
 # Make sure that the underlying container is patched to the latest versions
 RUN apt-get update && \
-    apt-get install -y gzip tar wget
+    apt-get install -y gzip tar wget && \
+    rm -rf /var/lib/apt/lists/*
```
Comment (hirusi) [inline docker/Dockerfile:6]: _Might_ be worth adding a note that any `apt-get install`s after this Docker step will require a new `apt-get update` before being able to install a package?

**Comment by hirusi** [comment]
Also worth noting that #168 has been approved already, so we might need to look at clean-ups specifically for Alpine in _that_ Docker image.

File: docker/Dockerfile (hunk 1 above)
Comment (PeterDaveHello) [reply docker/Dockerfile:6]: As there is no any other apt-get install yet, I prefer not to have so many comments in Dockerfile, and once another apt-get install was added, contributor will find out, right? :)

**Comment by jespino** [comment]
This is now fixed by the #168 PR

**Pull request closed without merging**